.gitignore

```diff
@@ -32,6 +32,9 @@ packages/
 **/.vs
 .factorypath
 .metals/*
+nbproject/
+nbactions.xml
+nb-configuration.xml
```

Comment:
thanks for the PR. can you please let me more about these changes?

Reply:
looks like these are for NetBeans. i'm ok with these.

Reply:
Yes I'm using Netbeans IDE. This is to exclude Netbeans metadata. It's comparible to `.idea` for Intelligj.